### PR TITLE
Add workshop schedule

### PR DESCRIPTION
**Pull request by lunson**
Adds workshop schedule from @pwalczysko's comment in https://trello.com/c/RIcmFL52/21-virtual-agenda#comment-5ebe75095270d6516ec87667

I agree that links should be added later when they're finalized. You can replace the "Details to follow soon" text with them as they become ready.

### CODE REVIEW

**Comment by lunson** [comment]
Added descriptions based on suggestions from @pwalczysko's comment in https://trello.com/c/RIcmFL52/21-virtual-agenda#comment-5ec2b68bfbb9717384550a98 

Please note that the optical matching requested can only work on desktop. It will break in mobile and possibly tablet due to the responsive nature of the layout; the only way it wouldn't is if all the workshop titles had the same character width.

**Comment by pwalczysko** [comment]
Thank you very much @lunson

**Comment by lunson** [comment]
@pwalczysko Please don't merge yet... Going to add one more visual cue to indicate repeating occurrences. Need a few minutes.

**Comment by pwalczysko** [comment]
> @pwalczysko Please don't merge yet... Going to add one more visual cue to indicate repeating occurrences. Need a few minutes.

Brilliant, understood. Take your time.

**Comment by pwalczysko** [comment]
@lunson Could you please indicate clearly that times are UK times ? Could you please also do the same for the first two day times ?
Thank you

**Comment by pwalczysko** [comment]
@lunson 

Please replace 
``Please note that with the exception of Large Import: Lessons from IDR, all the workshops have 2 occurences (1 occurence and 1 repeat).`` 
with 
``All the workshops will run twice except  Large Import: Lessons from IDR``

**Comment by pwalczysko** [comment]
``Learn more about using OMERO through our practical workshops``" --> ``Learn more about working with OME's tools and resources through our practical workshops``

Thank you

File: events/ome-community-meeting-2020/workshops/index.html [hunk 1]
```diff
@@ -3,19 +3,116 @@
...
+                    <p class="tiny-print">{{ agenda_item.description_a }}</p>
+                </div>
+                {% endif %}
+            </div>
+        </div>
+        <hr class="invisible">
+        {% endfor %}
+    </div>
+    <div class="small-6 medium-6 columns">
+        <div class="medium-12 columns"><h5 class="text-center ome-blue"><span class="hide-for-small-only">Workshop - </span>2nd parallel</h5></div>
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:100]:
```suggestion
        <div class="medium-12 columns"><h5 class="text-center ome-blue"><span class="hide-for-small-only">Workshop-</span>Stream 1</h5></div>
```

**Comment by pwalczysko** [comment]
Replace 

Workshop 1st Parallel -> ``Workshop-Stream 1``
Workshop 2nd Parallel -> ``Workshop-Stream 2``

File: events/ome-community-meeting-2020/workshops/index.html [hunk 2]
```diff
@@ -3,19 +3,116 @@
...
 title: Day 3 - Thursday, May 28
 subheader: Learn more about using OMERO through our practical workshops
+schedule:
+    - time_track_a: "11:00 - 13:00"
+      title_track_a: "Introduction to OMERO"
+      color_code_a: "#FFEBEE"
+      id_a: 1a
+      description_a: "Basic concepts, import, viewing and management of images and metadata in OMERO will be introduced. Data mining using OMERO.web and its extensions, such as OMERO.parade, will be shown. No previous knowledge about OMERO is assumed."
+      time_track_b: "11:00 - 13:00"
+      title_track_b: "OMERO systems installation and configuration"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:13]:
```suggestion
      title_track_b: "OMERO Systems Installation and Configuration"
```

File: events/ome-community-meeting-2020/workshops/index.html [hunk 3]
```diff
@@ -3,19 +3,116 @@
...
+      color_code_b: "#E3F2FD"
+      id_b: 1b
+      description_b: "Over the past few years we have updated our OMERO deployment instructions to follow modern deployment practices. These take advantage of automation tools such as Ansible and Docker to ensure you can reliably and reproducibly install and upgrade your OMERO.servers. This workshop will introduce you to some of the resources available, and will include some live demonstrations. This will be followed by time for a general discussion or questions, for instance on deployments such as the IDR."
+    - time_track_a: "13:00 - 15:00"
+      title_track_a: "Next Gen File Formats"
+      color_code_a: "#E0F2F1"
+      id_a: 2a
+      description_a: "From pyramidal OME-TIFFs to Zarr/n5 prototypes, we will walk through several recent and upcoming developments in our search for a next-general file format (NGFF). Demos include ImageJ macros, Bio-Formats command-line tools, an ome-zarr plugin for napari, and an OMERO microservice for publishing Zarr filesets."
+      time_track_b: "13:00 - 14:00"
+      title_track_b: "OMERO.figure with scripting"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:23]:
```suggestion
      title_track_b: "OMERO.figure with Scripting"
```

File: events/ome-community-meeting-2020/workshops/index.html [hunk 4]
```diff
@@ -3,19 +3,116 @@
...
+      color_code_b: "#FFEBEE"
+      id_b: 3b
+      description_b: "Basic concepts, import, viewing and management of images and metadata in OMERO will be introduced. Data mining using OMERO.web and its extensions, such as OMERO.parade, will be shown. No previous knowledge about OMERO is assumed."
+    - time_track_a: "\xA0"
+      title_track_a: "\xA0"
+      id_a:
+      description_a:
+      dirty_hack: yes
+      time_track_b: "17:00 - 18:00"
+      title_track_b: "OMERO.figure with scripting"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:43]:
```suggestion
      title_track_b: "OMERO.figure with Scripting"
```

File: events/ome-community-meeting-2020/workshops/index.html [hunk 5]
```diff
@@ -3,19 +3,116 @@
...
+      title_track_b: "Large Import: Lessons from IDR"
+      id_b: 5b
+      description_b: "Possibilities for importing large and heterogeneous data into OMERO will be explored. This includes command-line import, in-place import and bulk import."
+    - time_track_a: "21:00 - 23:00"
+      title_track_a: "Analysis with IDR"
+      color_code_a: "#ECEFF1"
+      id_a: 5a
+      description_a: "We will give an overview of the various analysis entry points. We will then focus on analysing data using the Python API. We will show how to set up a Conda environment and how to integrate with 3rd party Python analysis tools such as ilastik."
+      time_track_b: "21:00 - 23:00"
+      title_track_b: "OMERO systems installation and configuration"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/workshops/index.html:62]:
```suggestion
      title_track_b: "OMERO Systems Installation and Configuration"
```

**Comment by pwalczysko** [comment]
@lunson applied immediately some easy fixes of capitalization, see new commits from me

**Comment by pwalczysko** [comment]
Merging on the strength of https://trello.com/c/RIcmFL52/21-virtual-agenda#comment-5ec2bde20413a22023b603d8

Thank you again @lunson

**Pull request merged**